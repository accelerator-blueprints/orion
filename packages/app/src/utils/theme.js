export default {
  color: {
    blue: '#2165e5',
    midnightBlue: '#194cac',
    sand4: '#6d6d68',
    sand3: '#908a8a',
    sand2: '#a8a4a3',
    sand1: '#f4f4f2',
    supersplit: '#fd775e',
    brunchPink: '#fd7a9e',
    bubblegum: '#f9c3c0',
  },
  fontFamily: {
    header: 'Poppins, sans-serif',
    body: 'Didact Gothic, sans-serif',
  },
  breakpoint: {
    smUp: { min: '600px' },
    mdUp: { min: '960px' },
    lgUp: { min: '1200px' },
    xlUp: { min: '1920px' },
    smDown: { max: '600px' },
    mdDown: { max: '960px' },
    lgDown: { max: '1200px' },
    xlDown: { max: '1920px' },
  },
}
