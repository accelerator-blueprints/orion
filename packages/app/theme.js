/**
 * To switch themes, edit the lines below, and `themeName` in gatsby-config.js
 *
 * Available themes:
 *
 * 'nearform-theme'
 * 'efqm-theme'
 *
 **/
export { default } from 'efqm-theme'
export * from 'efqm-theme'
