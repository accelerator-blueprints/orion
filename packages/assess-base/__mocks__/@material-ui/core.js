import React from 'react'
const materialUi = jest.requireActual('@material-ui/core')

module.exports = {
  ...materialUi,
  // Mocking Button and IconButton prevents warnings when passing a Link component to them
  Button: jest.fn(props =>
    props.component ? (
      <a href={props.to}>{props.children}</a>
    ) : (
      <button {...props}>{props.children}</button>
    )
  ),
  IconButton: jest.fn(props =>
    props.component ? (
      <a href={props.to}>{props.children}</a>
    ) : (
      <button {...props}>{props.children}</button>
    )
  ),
}
