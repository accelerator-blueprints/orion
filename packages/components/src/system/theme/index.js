import { fontFamilies } from './typography'
import { OrangePalette, KnowledgebasePalette } from './colors'
import layout from './grid'

export const OrangeTheme = { ...fontFamilies, ...OrangePalette, ...layout }
export const KnowledgebaseTheme = {
  ...fontFamilies,
  ...KnowledgebasePalette,
  ...layout
}
