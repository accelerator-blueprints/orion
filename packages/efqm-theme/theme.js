const fontFamily = (exports.fontFamily = 'Lato, sans-serif')

const typography = {
  h1: {
    fontFamily,
    fontWeight: '900',
    fontSize: 28,
    letterSpacing: 0.5,
  },
  h2: {
    fontFamily,
    fontWeight: 'bold',
    fontSize: 18,
    lineHeight: 'normal',
    letterSpacing: -0.1,
  },
  h3: {
    fontFamily,
    fontWeight: '900',
    fontSize: 12,
    letterSpacing: 1.8,
    textTransform: 'uppercase',
  },
  h4: {
    fontFamily,
    fontWeight: 'bold',
    fontSize: 11,
    letterSpacing: 1.2,
    textTransform: 'uppercase',
  },
  h5: {
    fontFamily,
    fontWeight: 'bold',
  },
  h6: {
    // Mainly used for bold numbers in tables and graphics
    fontFamily,
    fontSize: 20,
    fontWeight: '900',
    lineHeight: 1.4,
    letterSpacing: -0.2,
  },
  body2: {
    fontFamily,
    fontSize: 14,
    letterSpacing: 'normal',
  },
  body1: {
    fontFamily,
    fontSize: 16,
    letterSpacing: 'normal',
  },
  placeholder: {
    fontFamily,
    fontSize: 14,
    lineHeight: 'normal',
    letterSpacing: -0.1,
  },
  button: {
    fontFamily,
    fontSize: 12,
    fontWeight: '900',
    lineHeight: 'normal',
    letterSpacing: 1.8,
  },
}

const efqmDigitalPalette = {
  // Names / keys are unofficial descriptions
  slateGrey: 'rgb(69, 75, 102)',
  navyBlue: 'rgb(73, 98, 173)',
  cyan: 'rgb(80, 184, 242)',
  aqua: 'rgb(121, 204, 198)',
  emeraldGreen: 'rgb(112, 214, 112)',
  pearGreen: 'rgb(155, 220, 106)',

  // Shades used in designs not included in EFQM docs
  paleGrey: 'rgb(244, 246, 248)',
  midGrey: 'rgb(152, 175, 198)',
  white: '#fff',
  black: '#000',
}

const shadows = [
  'none',
  // material ui requires  25 elevations
  ...new Array(24).fill(
    '0 0 5px 0 rgba(0, 0, 0, 0.1), 0 2px 10px 0 rgba(0, 0, 0, 0.1)'
  ),
]

exports.muiTheme = {
  overrides: {
    MuiButton: {
      root: {
        whiteSpace: 'nowrap',
        boxShadow: shadows[1],
      },
      text: {
        boxShadow: 'none',
      },
      contained: {
        padding: '9px 24px 8px',
      },
      outlinedPrimary: {
        border: 'none',
        '&:hover': {
          border: 'none',
        },
      },
      outlinedSecondary: {
        border: 'none',
        '&:hover': {
          border: 'none',
        },
      },
      outlined: {
        padding: '8px 24px 7px',
      },
      sizeSmall: {
        fontSize: 11,
      },
    },
    MuiInput: {
      root: {
        ...typography.body1,
        borderRadius: 3,
        backgroundColor: efqmDigitalPalette.paleGrey,
      },
      input: {
        padding: '7.5px 12px 8.5px',
      },
    },
    MuiOutlinedInput: {
      root: {
        ...typography.body1,
        borderRadius: 3,
        boxShadow: shadows[1],
        background: efqmDigitalPalette.white,
      },
      notchedOutline: {
        border: 'none',
      },
      input: {
        padding: '7.5px 12px 8.5px',
      },
    },
    MuiTypography: {
      gutterBottom: {
        marginBottom: '.75em',
      },
    },
    MuiTableCell: {
      head: {
        ...typography.h4,
      },
      body: {
        ...typography.body1,
      },
    },
  },
  props: {
    MuiInput: {
      disableUnderline: true,
    },
    MuiPaper: {
      elevation: 1,
    },
  },
  typography: {
    fontFamily,
    fontWeight: 'normal',
    ...typography,
  },
  palette: {
    primary: {
      main: efqmDigitalPalette.navyBlue,
      light: efqmDigitalPalette.cyan,
      dark: efqmDigitalPalette.slateGrey,
    },
    secondary: {
      main: efqmDigitalPalette.aqua,
      light: efqmDigitalPalette.pearGreen,
      dark: efqmDigitalPalette.emeraldGreen,
    },
    background: {
      default: efqmDigitalPalette.white,
      paper: efqmDigitalPalette.white,
      light: efqmDigitalPalette.paleGrey,
      dark: efqmDigitalPalette.midGrey,
    },
    contrastThreshold: 1.6,
  },
  shadows,
}

exports.googleFonts = ['Lato:400,700,900']
