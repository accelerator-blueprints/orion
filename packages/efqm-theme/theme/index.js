const palette = require('./mui/palette')
const typography = require('./mui/typography')
const shadows = require('./mui/shadows')
const props = require('./mui/props')

const MuiButton = require('./mui/muiComponentsOverrides/MuiButton')
const MuiListItemText = require('./mui/muiComponentsOverrides/MuiListItemText')
const MuiOutlinedInput = require('./mui/muiComponentsOverrides/MuiOutlinedInput')
const MuiTypography = require('./mui/muiComponentsOverrides/MuiTypography')
const MuiTableCell = require('./mui/muiComponentsOverrides/MuiTableCell')
const MuiInput = require('./mui/muiComponentsOverrides/MuiInput')
const MuiInputBase = require('./mui/muiComponentsOverrides/MuiInputBase')

const articleTypography = require('./custom/articleTypography')
const articleStatusColor = require('./custom/articleStatusColor')
const articleStatusLabelColor = require('./custom/articleStatusLabelColor')
const articleStatusIconColor = require('./custom/articleStatusIconColor')
const articleRateIconColor = require('./custom/articleRateIconColor')
const articleWidgetColor = require('./custom/articleWidgetColor')
const taxonomyColor = require('./custom/taxonomyColor')
const editorsPicks = require('./custom/editorsPicks')
const iconLight = require('./custom/iconLight')
const profileSpacerBackground = require('./custom/profileSpacerBackground')

exports.muiTheme = {
  overrides: {
    MuiButton,
    MuiInput,
    MuiInputBase,
    MuiListItemText,
    MuiOutlinedInput,
    MuiTypography,
    MuiTableCell,
  },
  props,
  typography,
  palette,
  shadows,
  iconLight,
  articleTypography,
  articleStatusColor,
  articleStatusLabelColor,
  articleStatusIconColor,
  articleWidgetColor,
  articleRateIconColor,
  profileSpacerBackground,
  taxonomyColor,
  editorsPicks,
}

exports.googleFonts = ['Lato:400,700,900']
