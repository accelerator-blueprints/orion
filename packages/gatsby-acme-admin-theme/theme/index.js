const MuiListItemText = require('./mui/muiComponentsOverrides/mui-list-item-text')
const MuiIcon = require('./mui/muiComponentsOverrides/mui-icon')
const MuiDrawer = require('./mui/muiComponentsOverrides/mui-drawer')
const MuiPaper = require('./mui/muiComponentsOverrides/mui-paper')
const MuiAppBar = require('./mui/muiComponentsOverrides/mui-app-bar')
const MuiPopover = require('./mui/muiComponentsOverrides/mui-popover')
const MuiButton = require('./mui/muiComponentsOverrides/mui-button')
const MuiInput = require('./mui/muiComponentsOverrides/mui-input')

// Section below are theme key word specific overrides https://material-ui.com/customization/default-theme/
const palette = require('./mui/palette')
const typography = require('./mui/typography')

// Custom class related themes
const articleContent = require('./custom/article-content')
const articleMetadata = require('./custom/article-metadata')
const footer = require('./custom/footer')
const searchInput = require('./custom/search-input')
const secondaryAppBar = require('./custom/secondary-app-bar')

exports.muiTheme = {
  overrides: {
    MuiListItemText,
    MuiIcon,
    MuiDrawer,
    MuiPaper,
    MuiAppBar,
    MuiPopover,
    MuiButton,
    MuiInput,
  },
  palette,
  typography,
  articleContent,
  articleMetadata,
  footer: { footer },
  searchInput,
  secondaryAppBar,
}

exports.googleFonts = ['Titillium Web:400,700,900']
