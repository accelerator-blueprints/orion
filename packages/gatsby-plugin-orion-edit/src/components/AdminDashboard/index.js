import React from 'react'
import SideBarMenu from 'gatsby-plugin-orion-core/src/components/SideBarMenu'
import { makeStyles } from '@material-ui/core'

const drawerWidth = 318
const useStyles = makeStyles(theme => ({
  '@global': {
    'html, body, body > div:first-child, body > div:first-child > div:first-child, body > div:first-child > div:first-child > div:first-child': {
      height: '100%',
    },
    body: {
      overflowY: 'hidden',
    },
  },
  root: {
    backgroundColor: '#204c67',
    display: 'flex',
    height: '100%',
    overflow: 'hidden',
  },
  heading: {
    marginBottom: '22px',
    color: theme.palette.common.white,
    padding: '0 16px',
  },
  content: {
    backgroundColor: theme.palette.common.white,
    flex: 1,
    margin: 0,
    padding: theme.spacing(2),
    overflowY: 'scroll',
    position: 'relative',
  },
  main: {
    display: 'flex',
    flex: 1,
    flexDirection: 'column',
    marginLeft: -17,
    zIndex: 1,
  },
  side: {
    display: 'block',
    overflowY: 'scroll',
    marginBottom: theme.spacing(1),
    height: '100%',
    paddingLeft: '28px',
    fondWeight: '700',
    width: drawerWidth,
    '& > div': {
      width: drawerWidth,
    },
    '& ul': {
      height: '100%',
      padding: 0,
    },
  },
}))
const AdminDashboard = ({ data, heading }) => {
  const classes = useStyles()
  return (
    <div className={classes.root}>
      <div className={classes.side}>
        <h1 className={classes.heading}>{heading}</h1>
        <SideBarMenu data={data} depthIndent={20} path="/" />
      </div>
      <div className={classes.main}>
        <div className={classes.content}>
          <h1>hello</h1>
        </div>
      </div>
    </div>
  )
}

export default AdminDashboard
