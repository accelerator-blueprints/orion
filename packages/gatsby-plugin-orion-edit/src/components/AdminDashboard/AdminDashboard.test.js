import React from 'react'
import { render, within } from '@testing-library/react'
import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles'
import theme from 'gatsby-theme-acme'

import AdminDashboard from '.'

const mockData = [
  {
    label: 'Quick find',
    iconClass: 'fas fa-search',
    to: '/quick-find',
  },
  {
    label: 'Pages',
    to: '/pages',
    iconClass: 'fas fa-file',
  },
  {
    label: 'Articles',
    to: '/articles',
    iconClass: 'fas fa-newspaper',
  },
  {
    label: 'Menus',
    to: '/menus',
    iconClass: 'fas fa-list-alt',
  },
  {
    label: 'Categories / Tags',
    to: '/categories',
    iconClass: 'fas fa-tag',
  },
  {
    label: 'Media Library',
    to: '/media-library',
    iconClass: 'fas fa-photo-video',
  },
  {
    label: 'Users',
    to: '/users',
    iconClass: 'fas fa-user-friends',
  },
  {
    label: 'goto View',
    to: '',
    iconClass: 'fas fa-eye',
    onClick: () => {},
  },
  {
    label: 'Sign out',
    to: '/sign-out',
    iconClass: 'fas fa-long-arrow-alt-left',
  },
]
const mockContent = [
  {
    label: 'Pages',
    to: '/pages/create',
    Image: () => <div>mockPagesIcon</div>,
  },
  {
    label: 'Articles',
    to: '/pages/create',
    Image: () => <div>mockArticlesIcon</div>,
  },
]

const renderComponent = () =>
  render(
    <ThemeProvider theme={createMuiTheme(theme)}>
      <AdminDashboard data={mockData} content={mockContent} heading="test me" />
    </ThemeProvider>
  )

describe('AdminDashboard component', () => {
  it('renders a header', async () => {
    const { getByText } = renderComponent()
    expect(getByText('test me')).toBeInTheDocument()
  })
  it('renders a side bar with a list of links', async () => {
    const { container } = renderComponent()
    const { getByText } = within(
      container.querySelector('[class^=makeStyles-side]')
    )
    expect(getByText('Quick find')).toBeInTheDocument()
    expect(getByText('Pages')).toBeInTheDocument()
    expect(getByText('Articles')).toBeInTheDocument()
    expect(getByText('Menus')).toBeInTheDocument()
    expect(getByText('Categories / Tags')).toBeInTheDocument()
    expect(getByText('Media Library')).toBeInTheDocument()
    expect(getByText('Users')).toBeInTheDocument()
    expect(getByText('goto View')).toBeInTheDocument()
    expect(getByText('Sign out')).toBeInTheDocument()
  })

  it('renders a list of menu cards', () => {
    const { container } = renderComponent()
    const { getByText } = within(
      container.querySelector('[class^=makeStyles-main]')
    )
    expect(getByText('Pages')).toBeInTheDocument()
    expect(getByText('Articles')).toBeInTheDocument()
  })
})
