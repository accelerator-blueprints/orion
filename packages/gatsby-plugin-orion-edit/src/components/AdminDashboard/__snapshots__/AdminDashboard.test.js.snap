// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminDashboard component uses the MUI theme components and classes to style the page 1`] = `
<div>
  <div
    class="makeStyles-root-361"
  >
    <div
      class="makeStyles-side-365"
    >
      <h1
        class="makeStyles-heading-362"
      >
        test me
      </h1>
      <ul
        class="MuiList-root makeStyles-root-367 MuiList-padding"
      >
         
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/quick-find"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-search MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Quick find
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/pages"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-file MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Pages
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/articles"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-newspaper MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Articles
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/menus"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-list-alt MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Menus
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/categories"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-tag MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Categories / Tags
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/media-library"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-photo-video MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Media Library
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/users"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-user-friends MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Users
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <div
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          role="button"
          tabindex="0"
          to=""
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-eye MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              goto View
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </div>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
          href="/sign-out"
          role="button"
          tabindex="0"
        >
          <div
            style="padding-right: 0px;"
          />
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root fas fa-long-arrow-alt-left MuiIcon-colorPrimary"
          />
          <div
            class="MuiListItemText-root"
          >
            <span
              class="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
            >
              Sign out
            </span>
          </div>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
         
      </ul>
    </div>
    <div
      class="makeStyles-main-363"
    >
      <div
        class="makeStyles-top-366"
      />
      <ul
        class="MuiGridList-root makeStyles-gridList-364"
        style="margin: -2px;"
      >
        <div
          class="MuiPaper-root MuiCard-root makeStyles-root-432 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <button
            class="MuiButtonBase-root MuiCardActionArea-root"
            tabindex="0"
            type="button"
          >
            <a
              class="makeStyles-link-433"
              href="/pages"
            >
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiCardMedia-root makeStyles-media-434"
                  style="background-image: url(todo);"
                  title="Pages"
                />
                <h5
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-colorSecondary MuiTypography-alignCenter"
                >
                  Pages
                </h5>
              </div>
            </a>
            <span
              class="MuiCardActionArea-focusHighlight"
            />
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiPaper-root MuiCard-root makeStyles-root-432 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <button
            class="MuiButtonBase-root MuiCardActionArea-root"
            tabindex="0"
            type="button"
          >
            <a
              class="makeStyles-link-433"
              href="/articles"
            >
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiCardMedia-root makeStyles-media-434"
                  style="background-image: url(todo);"
                  title="Articles"
                />
                <h5
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-colorSecondary MuiTypography-alignCenter"
                >
                  Articles
                </h5>
              </div>
            </a>
            <span
              class="MuiCardActionArea-focusHighlight"
            />
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiPaper-root MuiCard-root makeStyles-root-432 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <button
            class="MuiButtonBase-root MuiCardActionArea-root"
            tabindex="0"
            type="button"
          >
            <a
              class="makeStyles-link-433"
              href="/menus"
            >
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiCardMedia-root makeStyles-media-434"
                  style="background-image: url(todo);"
                  title="Menus"
                />
                <h5
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-colorSecondary MuiTypography-alignCenter"
                >
                  Menus
                </h5>
              </div>
            </a>
            <span
              class="MuiCardActionArea-focusHighlight"
            />
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiPaper-root MuiCard-root makeStyles-root-432 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <button
            class="MuiButtonBase-root MuiCardActionArea-root"
            tabindex="0"
            type="button"
          >
            <a
              class="makeStyles-link-433"
              href="/users"
            >
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiCardMedia-root makeStyles-media-434"
                  style="background-image: url(todo);"
                  title="Users"
                />
                <h5
                  class="MuiTypography-root MuiTypography-h5 MuiTypography-colorSecondary MuiTypography-alignCenter"
                >
                  Users
                </h5>
              </div>
            </a>
            <span
              class="MuiCardActionArea-focusHighlight"
            />
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </ul>
    </div>
  </div>
</div>
`;
